src/utils/common.js

```diff
@@ -591,18 +591,26 @@ export const getDeviceWidth = () => {
   return Dimensions.get('window').width;
 };
 
-export const formatTransactionFee = (feeInWei: string | number, gasToken: ?GasToken): string => {
-  if (!feeInWei) return '';
+export const getFormattedTransactionFeeValue = (feeInWei: string | number | BigNumber, gasToken: ?GasToken): string => {
+  // fixes exponential values with BigNumber.toPrecision()
+  const parsedFeeInWei = typeof feeInWei === 'object' // BigNumber
```

Comment:
I think there is `Bignumber.IsBignumber()` method available

Reply:
will check and amend if works, thanks 👍